### ISSUE TITLE
General - Use Real Names for Model Formats

### ISSUE THREAD

**Issue by shaddatic** [X-Hax/SonicAdventureBlenderIO#40]
Although using the games they come from made sense in the beginning, as people become more familiarized with these formats it's starting to hinder more than it helps. For example, take the tooltip:
```
Ambient Color (SA2 Only).
```
With the current system, it is extremely ambiguous as to what this actually means, _especially_ to newcomers. In fact, I myself believed this was referring to SA2 the game for many years; however it's actually referring to the SA2 format, Chunk. It is of course possible to correct this to `(SA2 Format Only)`, however I don't think that is a long term solution to the problem.

Another issue is, by default, this naming scheme falls apart quickly: SA2B uses all 3 formats, in fact the most common format it uses is SA2 models, with the second most being SA1 models; and SADX and SA2 both use SA1 _and_ SA2 models. That alone confuses people far more than calling the formats by their actual names would; and I know that because I remember being very confused by it when I started.

So, perhaps it's time to switch to the formats' real names, eg:
```
Ambient Color (Chunk Only)
```
Not only would this clean up any ambiguous language, but it would also make it clear SAIO _can_ be used for games other than the Sonic Adventure series. Any potential questions for what game uses what can be cleared up in the SAIO Manual, or, at worst, a single question in a relevant chat/channel.

It would also open the door for more model file formats without needing to keep `.sa#mdl` front and center. For example, a native Ninja Ascii (NJA/NAM/TLS) exporter for SAIO would be very useful for me and a few other creators that mainly use that file format, however accessing it by selecting the `SA2 Model (.sa2mdl)` export option is a little odd since I have no intention of exporting using `.sa2mdl`; the same issue exists with the current NJ file export. Meanwhile, `Chunk (eg. sa2mdl)` makes it more clear we're exporting the Chunk model format in a range of different file formats, and not just one.

In addition, SA Tools has already switched to the real names, meaning there's currently no unified language across the tools. It's possible to extract a 'Chunk' model with SAMDL and export it, then import an 'SA2 Model' into SAIO.

I do understand that this change does have the potential to confuse people at first, especially since it's stood for so long, and that should be taken into account before any plans are made. However, in my experience, most relatively adept SAIO users understand that Basic is the SA1 format, Chunk is the SA2 format, and Ginja is the SA2B format; to the extent that they use these names in regular conversation So, a base already exists that can smooth the transition.

TL;DR: I am proposing we switch from the old `SA1 Model`/`SA2 Model`/`SA2B Model` naming scheme to the actual names for these formats used by Ninja and SA Tools, eg. `Basic`/`Chunk`/`Ginja`. I believe doing so will resolve a lot of confusion and allow for more concise language in option names and tooltips.

**Comment by PiKeyAr** [comment]
Small addition with regard to `Basic`: there's the original Ninja Basic format used in SA1 DC, SADX GC and SA2 DC, and another one with extended Meshset and Model structs, which is used in the 2004 PC port and Steam/X360/PS3 versions of SADX. It was called BasicDX or Basic (SADX) in the tools originally but I suggest calling it `Basic+` to avoid confusion. 

https://github.com/X-Hax/sa_tools/wiki/Working-with-Binary-Files#basic-format-sa1-format


`Ginja` is fine if we agree it's the hybrid format used in SA2B and Billy Hatcher, and not "Ninja Gamecube", which could be called `Ninja Chunk (Big Endian)`, though that one also has at least two variations.
